packages/@react-aria/tag/src/TagKeyboardDelegate.ts

```diff
@@ -86,6 +96,50 @@ export class TagKeyboardDelegate<T> extends GridKeyboardDelegate<T, GridCollecti
       if (this.focusMode === 'row') {
         return key;
       }
+    } else {
+      return this.getLastKey();
```

Comment:
There's a bug here, if you go to https://reactspectrum.blob.core.windows.net/reactspectrum/3fcf89a85757b0d5b352e20e1c3eac3c9b55d24e/storybook/index.html?path=/story/taggroup--disabledkeys
you can focus the disabled key by wrapping around from the end.

The logic should probably be more like https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/tabs/src/TabsKeyboardDelegate.ts